### DISCUSSION TITLE
mesa-frames 0.1.0 stable release

### DISCUSSION THREAD

**Discussion by EwoutH** [projectmesa/mesa-frames#120] [General]
Hi @adamamer20, how are things?

I would like to discuss an initial "official" release of 0.1.0. It seems nice to have it out there, that it can be easily pip installed etc.

This week I have time to help and support!

**Comment by adamamer20** [comment]
I think five things are the priorities before release:

1. Deprecate and remove AgentSetPandas (prioritizing polars-based implementation)
2. Implement proper CI/CD pipeline
3. Expand documentation with more in-depth explanations and examples
4. Align functionality with core Mesa package (excluding visualization components for now)
5. Implement concurrency handling instead of leaving it to the user
